.changeset/soft-planets-watch.md

```diff
@@ -0,0 +1,6 @@
+---
+"@redocly/openapi-core": patch
+"@redocly/cli": patch
+---
+
+Fixed an issue where `$ref`s ending in `#` (instead of `#/`) would break the application.
```

Comment:
Do we silently handle this situation, or provide a better error message when we see it?